src/main/java/com/google/sps/data/Country.java

```diff
@@ -282,4 +287,10 @@ private Country(String longName) {
   public String getLongName() {
     return longName;
   }
+
+  public static Collection<Country> valuesSorted() {
+    List<Country> values = Arrays.asList(Country.values());
```

Comment:
doesn't make sense to sort static values every time (un-necessary computation) - can we instead store all the values in sorted order and write a test to make sure its already sorted (so that if anyone messes around with the values that test will fail - and you have a way to check that the statically stored data is already conformant)